scripts/desktop-launch.sh

```diff
@@ -1,8 +1,12 @@
-#!/bin/sh
+#!/bin/bash
 
-exec gdbus call \
-     --session \
-     --dest io.snapcraft.Launcher \
-     --object-path /io/snapcraft/PrivilegedDesktopLauncher \
-     --method io.snapcraft.PrivilegedDesktopLauncher.OpenDesktopEntry \
-     "$1"
```

Comment:
Mmm... This script is already being used from `scripts/make-launchers.sh`. Removing this maybe could make it fail...